platform-manageability-agent/cmd/platform-manageability-agent/platform-manageability-agent.go

```diff
@@ -10,22 +10,31 @@ import (
 	"flag"
 	"fmt"
 	"os"
+	"os/exec"
 	"os/signal"
 	"sync"
 	"sync/atomic"
 	"syscall"
 	"time"
 
 	"github.com/cenkalti/backoff/v4"
+	"github.com/labstack/gommon/log"
```

Comment:
Why is this log package being imported? Each agent uses the common log functionality which is based on the logrus package imported after this. PMA should use the same, not a different log package